01_data_exploration.ipynb

```diff
@@ -10,17 +10,17 @@
     "# 1. Data Exploration\n",
```

Comment:
<p>Could: Phrase the learning outcomes in Imperative sentences.</p><p> </p><p>The second outcome is incomplete&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/informatics-lab/ml_weather_tutorial/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>